### ISSUE TITLE
README has the wrong dependency name

### ISSUE THREAD

**Issue by dragossusi** [TimOrtel/GRPC-Kotlin-Multiplatform#18]
Looks like README.md has the wrong dependency name:

```kotlin
api("com.github.TimOrtel.GRPC-Kotlin-Multiplatform:grpc-multiplatform-lib:<latest version>")
```

I tried to add this to my app, but it did not find it for some reason.

```Could not find com.github.TimOrtel.GRPC-Kotlin-Multiplatform:grpc-multiplatform-lib:0.2.1.```

**Comment by johanesriandy** [comment]
Hi, i'd encountered the same issue but it worked with version `1.1.0`. I've checked the repo in Jitpack and it seems that the build was failed for later version

<img width="576" alt="image" src="https://user-images.githubusercontent.com/46487225/223388794-c732f134-daef-4911-8042-b94f63caf6b5.png">

**Comment by TimOrtel** [comment]
This should be fixed with version 0.3.0. Closing this for now.
